talkies.lua

```diff
@@ -310,7 +312,7 @@ function Talkies.draw()
   -- Message options (when shown)
   if currentDialog:showOptions() and currentMessage.complete then
     if currentDialog.inlineOptions then
-      local optionsY = textY+currentDialog.font:getHeight(currentMessage.visible)-(currentDialog.padding/1.6)
+      local optionsY = textY+currentDialog.font:getHeight(currentMessage.visible)-- -(currentDialog.padding/1.6)
```

Comment:
Why did you comment out the padding?

Reply:
Padding causes text overlap between first option and menu message, on erogodic example which uses the library with 24-sized font is not much noticeable
![Captura-210519-1153](https://user-images.githubusercontent.com/23223494/118853824-cafcbc00-b899-11eb-81d0-3a8f9bbcf150.PNG)
     _(Main Branch) first option is closer to menu message_

![Captura-210519-1154](https://user-images.githubusercontent.com/23223494/118854022-06978600-b89a-11eb-968e-8df8444ebba6.PNG)
    _(Pull Request) now is even_

Now, the issue occurs with low-sized fonts (this time with size=12)
![Captura-210519-1155](https://user-images.githubusercontent.com/23223494/118854348-4494aa00-b89a-11eb-88d9-d90a40c3e2c9.PNG)
    _(Main Branch) overlapped menu message and first option_

![Captura-210519-1156](https://user-images.githubusercontent.com/23223494/118854504-6a21b380-b89a-11eb-839b-bad986fca97f.PNG)
    _(Pull Request) now fixed_